workspace/TS100/src/syscalls.c

```diff
@@ -40,7 +40,10 @@ int _kill(int pid, int sig)
 void _exit (int status)
 {
 	_kill(status, -1);
-	while (1) {}		/* Make sure we hang here */
+	
+	/* devblackhunter: If you want to create a loop, don`t use a while(true),
+	its a fucking dumb */
```

Comment:
Seriously? comments should be worded in a manner that is not insulting to others.
Secondly, this call never makes its way into the binary.
Thirdly, Most compilers are aware of this and you end up with the same asm anyway.